Cargo.toml

```diff
@@ -13,17 +13,17 @@ categories = ["web-programming::http-client"]
 bytes = "0.4"
 futures = "0.1.15"
 hyper = "0.11"
-hyper-tls = "0.1.2"
+hyper-tls = { git = "https://github.com/scottschroeder/hyper-tls", branch = "noverify" }
 libflate = "0.1.11"
 log = "0.3"
 mime_guess = "2.0.0-alpha.2"
-native-tls = "0.1.3"
+native-tls = { git = "https://github.com/scottschroeder/rust-native-tls", branch = "noverify" }
```

Comment:
I'll update the `Cargo.toml` if/when new crates gets published with the proper code/dependencies.